app/javascript/components/public_health/PatientsTable.js

```diff
@@ -493,6 +494,25 @@ class PatientsTable extends React.Component {
     return moment(endOfMonitoring, 'YYYY-MM-DD').format('MM/DD/YYYY');
   }
 
+  formatLatestReport = data => {
+    const rowData = data.rowData;
+    return (
+      <Row>
+        <Col xs="1">
+          {rowData.latest_report.symptomatic && (
+            <span data-for={`${rowData.id.toString()}-symptomatic-icon`} data-tip="">
+              <i className="fas fa-exclamation-triangle red-icon"></i>
+              <ReactTooltip id={`${rowData.id.toString()}-symptomatic-icon`} multiline={false} place="left" type="dark" effect="solid">
+                <span>Monitoree&apos;s latest report was symptomatic</span>
```

Comment:
Does this cause some kind of linting error?
```
<span>Monitoree's latest report was symptomatic</span>
```

Reply:
Yes it does which is why I switched to use `&apos;`

Reply:
Got it, I'm wondering if
```
<span>{`Monitoree's latest report was symptomatic`}</span>

```
would work

Reply:
That works and there is no linting error, so I can go with that. Thanks for the tip!

Reply:
Addressed in [BAF4994](https://github.com/SaraAlert/SaraAlert/pull/745/commits/baf49948ca64a2c6eb3865faf010f2404e5d9b0d)